File: Front/ejemplo1/src/app/Modelo/Logistica.ts
import { NodeCompatibleEventEmitter } from "rxjs/internal/observable/fromEvent";

export class Logistica {
   id:string;
   tipoProducto:string;
   cantidadProducto:number;
   fechaRegistro: String;
   fechaEntrega: String;
   fecha: String;
   puertoEntrega:String;
   placaVehiculo:String;
   numeroFlota: string;
   numeroGuia: string;
   precioEnvio:number;
   descuentoOtorgado:number;
   /*
   
   
  
   */
   constructor(
    id: string,
    tipoProducto:string,  
    cantidadProducto:number,
    fechaRegistro:String,
    fechaEntrega:String,
    fecha:String,
    puertoEntrega: string,
    placaVehiculo: string,
    numeroFlota: string,
    numeroGuia: string,
    precioEnvio:number,
    descuentoOtorgado:number
    /*
    
    
   
*/
    ) {
        this. id=id; 
        this.tipoProducto=tipoProducto;
        this.cantidadProducto=cantidadProducto;
        this.fechaRegistro=fechaRegistro;
        this.fechaEntrega=fechaEntrega;
        this.fecha=fecha;
        this.puertoEntrega=puertoEntrega;
        this.placaVehiculo=placaVehiculo;
        this.numeroFlota=numeroFlota;
        this.numeroGuia=numeroGuia;
        this.precioEnvio=precioEnvio;
        this.descuentoOtorgado=descuentoOtorgado;
        /*
       
        
        
        */
      }
    }
   

File: BackConToken/pruebasIngeneo/domain/model/src/main/java/co/com/s/model/unitmeasure/Logistica.java
package co.com.s.model.unitmeasure;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
@NoArgsConstructor
@AllArgsConstructor
@Data
@Builder(toBuilder = true)
public class Logistica {

    private String id;
    private String tipoProducto;
    private Integer cantidadProducto;
    private LocalDateTime fechaRegistro;
    private LocalDateTime fechaEntrega;

    private String puertoEntrega;
    private String placaVehiculo;
    private String numeroFlota;
    private String numeroGuia;
    private Integer precioEnvio;
    private Integer descuentoOtorgado;

    /*



*/
}

